release notes/v0.47.0.md

```diff
@@ -64,6 +64,8 @@ console.log(context.cookies.length); // 0
 _Format as `<number> <present_verb> <object>. <credit>`_:
 
 - [#3338](https://github.com/grafana/k6/pull/3338) and [xk6-grpc#48](https://github.com/grafana/xk6-grpc/pull/48). Adds a support of the v1 of the gRPC reflection protocol.
+- [browser#1035](https://github.com/grafana/xk6-browser/pull/1035) Refactor `int64` timeout to `time.Duration` which ensures no confusion as to whether a timeout is in milliseconds or seconds.
```

Comment:
Does this change UX or it's an internal improvement?

Reply:
Hmm, not sure. It might be better placed in fix now that i think about it.

Before the change a timeout set by the user would be converted to second, when they expected it to be in milliseconds. WDYT?

Reply:
We should also mention that our documentation and code needed alignment. Now it is. So, I believe it's a bug fix.

Reply:
Resolved in a9cdddb154d2c584e5c34b9265e56997a223ac31